tests/python/relax/test_transform_legalize_ops_manipulate.py

```diff
@@ -888,5 +888,214 @@ def main(x: R.Tensor(("a", "b", "c"), "float32")) -> R.Tensor(("b", 1), "float32
     tvm.ir.assert_structural_equal(mod, Expected)
 
 
+def test_repeat():
+    # fmt: off
+    @I.ir_module
+    class Repeat:
+        @R.function
+        def main(x: R.Tensor((3, 2, 3), "float32")):
```

Comment:
I ignored some annotations in the Before module. The problem is, if we specify type annotations, `InferStructInfo` checking will be skipped. Then some errors in hand-written struct_infos may not be found.

For example, `test_nll_loss` in `test_transform_legalize_ops_nn.py

Reply:
What about adding this feature:

Always check struct_infos for operators, tuples, tuplegetitems, etc. when normalizing, even though it is specified?

It seems not hard to implement for us

@MasterJH5574